{'self': <unittest.main.TestProgram object at 0x102996150>, 'module': None, 'defaultTest': None, 'argv': None, '                                             testRunner': None, 'testLoader': <unittest.loader.TestLoader object at 0x102dcf050>, 'exit': True, 'verbosity': 1, 'failfast': None, 'catchbreak': None, 'buffer': None, 'warnings': None, 'tb_locals': False}
{'self': <unittest.main.TestProgram object at 0x104521190>, 'module': None, 'defaultTest': None, 'argv': ['tests_e2e/is_palindrome/test_is_palindrome.py'], 'testRunner': None, 'testLoader': <unittest.loader.TestLoader object at 0x104885950>, 'exit': True, 'verbosity': 1, 'failfast': None, 'catchbreak': None, 'buffer': None, 'warnings': None, 'tb_locals': False}
